src/inject.js

```diff
@@ -41,13 +47,17 @@ const injectStyleTag = (cssContents /* : string */) => {
             styleTag.type = 'text/css';
             styleTag.setAttribute("data-aphrodite", "");
             head.appendChild(styleTag);
+        } else if (styleTag.styleSheet) {
+            // $FlowFixMe: legacy Internet Explorer compatibility
+            styleString = styleTag.innerText;
         }
     }
 
 
     if (styleTag.styleSheet) {
+        styleString += cssContents;
         // $FlowFixMe: legacy Internet Explorer compatibility
-        styleTag.styleSheet.cssText += cssContents;
+        styleTag.styleSheet.cssText = styleString;
```

Comment:
I wonder if you could avoid managing `styleString` by doing this instead?

```js
styleTag.styleSheet.cssText = styleTag.innerText + cssContents;
```

With your current approach I mostly worry about something else modifying the stylesheet and then that getting blown away (in addition to the extra bookkeeping complexity). Doesn't seem super likely, but if we can trivially avoid it might as well, right?